src/include/daos_srv/rebuild.h

```diff
@@ -26,13 +26,15 @@ typedef enum {
 	RB_OP_REINT,
 	RB_OP_EXTEND,
 	RB_OP_RECLAIM,
+	RB_OP_FAIL_RECLAIM,
```

Comment:
in this patch only see the places check RB_OP_FAIL_RECLAIM, did not see where with set RB_OP_FAIL_RECLAIM?

Reply:
in rebuild_task_complete_schedule(), if rebuild fail, it will use FAIL_RECLAIM to cleanup the objects, instead of RECLAIM, which is used to cleanup the objects after reintegration.